packages/Ludown/docs/commands.md

```diff
@@ -141,6 +141,7 @@ To bootstrap translations of the language understanding content for your bot, yo
     -n, --lu_File <LU_File>          [Optional] Output .lu file name
     -c, --transate_comments          [Optional] Translate comments in .lu files
     -u, --translate_link_text        [Optional] Translate URL or .lu file reference link text
+    -x, --batch_translate <linesToBatch>  [Optional] Batch up <x> (1-25) number of lines before calling translation API. Defaults to 25.
```

Comment:
How about "-b" for match. using "-x" seems weird.

Reply:
Yes. switched to -b.